docs/conf.py

```diff
@@ -22,7 +22,7 @@
 
 project = "pyscal"
 author = u"Håvard Berland"
-copyright = "Equinor 2019-2020"
+copyright = "Equinor 2019-2021"
```

Comment:
```suggestion
copyright = "Equinor 2019-{datetime.date.today().year}"
```
Easier to maintain? 🕐